inference-engine/src/transformations/src/transformations/common_optimizations/moc_transformations.cpp

```diff
@@ -79,6 +81,8 @@ bool ngraph::pass::MOCTransformations::run_on_function(std::shared_ptr<ngraph::F
     if (!m_use_shapes) {
         manager.register_pass<ngraph::pass::DisableShapeOfConstantFolding>();
     }
+    manager.register_pass<ov::pass::RemoveConcatZeroDimInput>();
```

Comment:
Please put a comment why this transformation is located at the beginning of pipeline.

Reply:
done